docs/guides/python/multiple_threads.md

```diff
@@ -6,7 +6,7 @@ title: Multiple Python Threads
 This page demonstrates how to simultaneously insert into and read from a DuckDB database across multiple Python threads. 
 This could be useful in scenarios where new data is flowing in and an analysis should be periodically re-run. 
 Note that this is all within a single Python process (see the [FAQ](/faq) for details on DuckDB concurrency).
-Feel free to follow along in this [Google Collaboratory Notebook](https://colab.research.google.com/drive/190NB2m-LIfDcMamCY5lIzaD2OTMnYclB?usp=sharing).
+Feel free to follow along in this [Google Colab notebook](https://colab.research.google.com/drive/190NB2m-LIfDcMamCY5lIzaD2OTMnYclB?usp=sharing).
```

Comment:
> Colab, or "Colaboratory", allows you to write and execute Python in your browser

From https://colab.research.google.com

Reply:
Yes, both Colab and Colaboratory work but coLLaboratory with two L is definitely incorrect.

Reply:
Oops yes